source/Ship.cpp

```diff
@@ -2404,7 +2427,10 @@ bool Ship::IsReadyToJump(bool waitingIsReady) const
 			return false;
 	}
 
-	return true;
+	// For any ship that is not the player flagship,
+	// jumps should not be restricted by animation if they are not in the system.
+	return (GetState() == Body::BodyState::JUMPING && ReadyForAction()) ||
```

Comment:
```suggestion
	return (GetState() == BodyState::JUMPING && ReadyForAction()) ||
```
Would this work? I mean, a `Ship` is a `Body`.

Reply:
Seems to work!